.github/workflows/ci.yml

```
@@ -53,7 +53,7 @@ jobs:
           echo "Outputting all files in the packaging directory:"
           $contents = Get-ChildItem -Recurse -Path packaging | Select-Object -Property FullName
           echo $contents | Out-String -Width 1000
-          if (($contents | Measure-Object).Count -ne 6) {
+          if (($contents | Measure-Object).Count -ne 7) {
```

So others aren't as confused as we were
```suggestion
          # Directory entry, Metadata.ps1, Octopus-generated nuspec, + 2 assets & 2 nupkg
          if (($contents | Measure-Object).Count -ne 7) {
```